src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2255,3 +2255,10 @@ Reveal\ in\ file\ explorer=Reveal in file explorer
 Reset=Reset
 Reset\ entry\ types\ and\ fields\ to\ defaults=Reset entry types and fields to defaults
 This\ will\ reset\ all\ entry\ types\ to\ their\ default\ values\ and\ remove\ all\ custom\ entry\ types=This will reset all entry types to their default values and remove all custom entry types
+
+Replace\ tabs\ with\ space=Replace tabs with space
+Replace\ tabs\ with\ space\ in\ the\ field\ content.=Replace tabs with space in the field content.
+Remove\ redundant\ spaces=Remove redundant spaces
+Replaces\ consecutive\ spaces\ with\ a\ single\ space\ in\ the\ field\ content.=Replaces consecutive spaces with a single space in the field content.
+Remove\ digits=Remove digits
+Removes\ digits.=Removes digits.
```

Comment:
Could you remove the `.` at the end? This would reduce load at our translators (as it is the same string as in the line above).

(Hope, I did not overlook something)

Reply:
I added the full stop to make the formatter `getDescription()` consistent with other implementations of `Formatter`, which also have a full-stop at the end. Perhaps I should add a bit more to the description anyway, to make it more descriptive..

If not, I guess I should also make it `remove` instead of `remove**s**`, to make it the exact same string?

Reply:
Ah, didn't see the `s`. 😇  Then, everything is fine.